server/src/api/v2/polls/delete-poll.ts

```diff
@@ -0,0 +1,48 @@
+import {Request, Response} from 'express';
+import {wrapResponse} from '../../../functions/response-wrapper';
+import {Poll} from '../../../models/poll.model';
+import {PollAnswer} from '../../../models/poll-answer.model';
+
+export async function deletePoll(req: Request, res: Response): Promise<Response> {
+    let success = true;
+    //TODO Authoriaztion check, if polls can get created by no-admins
+    await Poll.update(
+        {
+            is_active: false,
+        },
+        {
+            where: {
+                id: req.params.id,
+                is_active: true
+            }
+        })
+        .catch(() => {
+            success = false;
+            return null;
+        });
+    if (!success) {
+        return res.status(500).send(wrapResponse(false, {error: 'Could not deactivate poll with id ' + req.params.id}));
+    }
+
+    await PollAnswer.update(
+        {
+            is_active: false,
+        },
+        {
+            where: {
+                poll_id: req.params.id,
+                is_active: true
+            }
+        })
+        .catch(() => {
+            success = false;
+            return null;
+        });
+    if (!success) {
+        return res.status(500).send(wrapResponse(false, {error: 'Could not deactivate pollanswer with id ' + req.params.id}));
+    }
+
```

Comment:
Sowas in einer Transaktion machen https://sequelize.org/master/manual/transactions.html, damit der Datenstand konsistent ist

Reply:
@TomasKostadinov @DominikRuf TODO, because "Sequelize does not use transactions by default. However, for production-ready usage of Sequelize, you should definitely configure Sequelize to use transactions."